INSTALL.md

```diff
@@ -2,111 +2,93 @@
 
 Below, we detail how to install Darts using either `conda` or `pip`.
 
+## From PyPI
+Install darts with all models expect the ones from optional dependencies (Prophet, LightGBM, CatBoost, see more on that [here](#enabling-optional-dependencies)): `pip install darts`.
+
+If this fails on your platform, please follow the official installation 
+guide for [PyTorch](https://pytorch.org/get-started/locally/), then try installing Darts again.
+
+As some dependencies are relatively big or involve non-Python dependencies,
+we also maintain the `u8darts` package, which provides the following alternate lighter install options:
+
+* Install darts with all available models: `pip install u8darts[all]`
+* Install core only (without neural networks, Prophet, LightGBM and Catboost): `pip install u8darts`
+* Install core + Prophet + LightGBM + CatBoost: `pip install "u8darts[notorch]"`
+* Install core + neural networks (PyTorch): `pip install "u8darts[torch]"` (equivalent to `pip install darts`)
+
 ## From conda-forge
 Currently only the x86_64 architecture with Python 3.8-3.10
 is fully supported with conda; consider using PyPI if you are running into troubles.
 
-To create a conda environment for Python 3.9
+Create a conda environment (e.g., for Python 3.10):
 (after installing [conda](https://docs.conda.io/en/latest/miniconda.html)):
 
-    conda create --name <env-name> python=3.9
+    conda create --name <env-name> python=3.10
 
-Don't forget to activate your virtual environment
+Activate the environment
 
     conda activate <env-name>
 
-As some models have relatively heavy dependencies, we provide two conda-forge packages:
+As some models have relatively heavy dependencies, we provide four conda-forge packages:
 
-* Install darts with all available models (recommended): `conda install -c conda-forge -c pytorch u8darts-all`.
+* Install darts with all available models: `conda install -c conda-forge -c pytorch u8darts-all`
+* Install core only (without neural networks, Prophet, LightGBM and Catboost): `conda install -c conda-forge u8darts`
+* Install core + Prophet + LightGBM + CatBoost: `pip install "u8darts[notorch]"`
```

Comment:
to be coherent with the others, should this flavor of the package be called `udarts-notorch`?

Reply:
👍